plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -366,6 +389,10 @@ export default defineComponent({
       type: Array,
       required: true,
     },
+    extensions: {
```

Comment:
See the comment below for the calculated method, if we wen't with the approach of obtaining the extra components via an event, we don't need the property at all.